RecoTracker/SpecialSeedGenerators/src/OutsideInMuonSeeder.cc

```diff
@@ -178,10 +184,19 @@ OutsideInMuonSeeder::produce(edm::Event & iEvent, const edm::EventSetup & iSetup
             }
         }
         if (tk.eta() > minEtaForTEC_) {
-            int iLayer = 9, found = 0;
-            std::vector< ForwardDetLayer const* > const & tec = measurementTracker->geometricSearchTracker()->posTecLayers();
-            for (auto it = tec.rbegin(), ed = tec.rend(); it != ed; ++it, --iLayer) {
-                if (debug_) std::cout << "\n ==== Trying TEC " << +iLayer << " ====" << std::endl;
+            auto forwLayers = measurementTracker->geometricSearchTracker()->posTecLayers();
```

Comment:
no, at most should be something like
auto forwLayers = tmpTkGeometry->isThere(GeomDetEnumerators::P2OTEC) ?
             measurementTracker->geometricSearchTracker()->posTidLayers()
           : measurementTracker->geometricSearchTracker()->posTecLayers();
(to be verified if const & compiles in such a case)